lib/config.js

```diff
@@ -136,6 +136,10 @@ config.views.vars = {
   //   email: {..., email: '...'}
   contact: {},
   contextUrls: {},
+  CONTEXT_MAP: {
```

Comment:
It looks like nothing else is capitalized here. This should therefore be changed to "contextMap".